tests/linux_kernel/helpers/test_slab.py

```diff
@@ -120,3 +123,55 @@ def test_slab_cache_for_each_allocated_object(self):
                 ),
                 [objects[i] for i in range(5)],
             )
+
+    @skip_unless_have_full_mm_support
+    @skip_unless_have_test_kmod
+    def test_find_containing_slab_cache(self):
+        cache = self.prog["drgn_test_kmem_cache"]
+        objects = self.prog["drgn_test_slab_objects"]
+
+        if self.prog["drgn_test_slob"]:
+            for obj in objects:
+                self.assertEqual(
+                    find_containing_slab_cache(self.prog, obj.value_()),
+                    NULL(self.prog, "struct kmem_cache *"),
+                )
+
+                self.assertEqual(
+                    find_containing_slab_cache(self.prog, obj),
```

Comment:
Let's test the other calling convention:

```suggestion
                    find_containing_slab_cache(obj),
```

Reply:
Resolved!